invenio_deposit/config.py

```diff
@@ -33,7 +33,7 @@
 DEPOSIT_SEARCH_API = '/api/deposits'
 """URL of search endpoint for deposits."""
 
-DEPOSIT_RECORDS_API = '/api/deposits/{pid_value}'
```

Comment:
This is a valid formatting string. Can you provide a use-case for which you would like to change it?